client/app/telecom/pack/xdsl/access/pack-xdsl-access.controller.js

```diff
@@ -1,171 +1,169 @@
-        $templateCache.put("pack-xdsl-access-tooltip-dslam.html", "<div class=\"tooltip-description\" data-translate=\"xdsl_access_dslam_reset_description\"></div><div class=\"text-warning\" data-translate=\"xdsl_access_dslam_reset_warning\"></div>");
-        $templateCache.put("pack-xdsl-access-tooltip-lnsApply.html", "<div class=\"tooltip-description\" data-translate=\"xdsl_access_lns_ratelimit_description\"></div><div class=\"text-warning\" data-translate=\"xdsl_access_lns_ratelimit_warning\"></div>");
-        $templateCache.put("pack-xdsl-access-tooltip-lns.html", "<div class=\"tooltip-description\" data-translate=\"xdsl_access_lns_description\"></div><div class=\"text-warning\" data-translate=\"xdsl_access_lns_warning\"></div>");
-        $templateCache.put("pack-xdsl-access-tooltip-deconsolidation.html", "<div class=\"tooltip-description\" data-ng-bind=\" ('xdsl_access_deconsolidation_warning_' + XdslAccess.lineDetails.deconsolidation) | translate\"></div>");
-        $templateCache.put("pack-xdsl-access-tooltip-ipDelete.html", "<div class=\"tooltip-description\" data-translate=\"xdsl_details_ips_remove_only_extra\"></div>");
-        $templateCache.put("pack-xdsl-access-tooltip-ips.html", "<div class=\"tooltip-description\" data-translate=\"xdsl_access_ipv6_description\"></div><div class=\"text-warning\" data-translate=\"xdsl_access_ipv6_warning\"></div>");
-        $templateCache.put("pack-xdsl-access-tooltip-dslamProfile.html", "<div class=\"text-left\"><p data-translate=\"xdsl_access_profile_tooltip_interleaved\"></p><p data-translate=\"xdsl_access_profile_tooltip_fast\"></p><p data-translate=\"xdsl_access_profile_tooltip_ginp\"></p><p data-translate=\"xdsl_access_profile_tooltip_auto\"></p><p data-translate=\"xdsl_access_profile_tooltip_snr\"></p><p class=\"text-warning\" data-translate=\"xdsl_access_profile_tooltip_time\"></p></div>");
-        /* eslint-enable max-len */
-    }
-
-    function setStatusLabel (status) {
-        switch (status) {
-        case "active":
-            self.statusLabel = "<i class=\"ovh-font ovh-font-success text-success fs16 right-space-m8\"></i> " + $translate.instant("xdsl_details_status_" + status);
-            break;
-        case "doing":
-        case "migration":
-        case "upgradeOffer":
-            self.statusLabel = "<i class=\"ovh-font ovh-font-success text-success fs16 right-space-m8\"></i> " + $translate.instant("xdsl_details_status_" + status);
-            break;
-        case "cancelled":
-        case "close":
-        case "deleting":
-        case "slamming":
-            self.statusLabel = "<i class=\"ovh-font ovh-font-failure text-danger fs16 right-space-m8\"></i> " + $translate.instant("xdsl_details_status_" + status);
-            break;
-        default :
-            self.statusLabel = status;
+angular.module("managerApp").controller("XdslAccessCtrl",
```

Comment:
Do you have any reason of modifying this file? I think that `grunt eslint` will warn you now!